components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -738,11 +743,41 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *provi
 func NewUpdateProcessingQueue(ctx context.Context, manager *update.Manager, workersAmount int, db storage.BrokerStorage, inputFactory input.CreatorForPlan,
 	provisionerClient provisioner.Client, publisher event.Publisher, logs logrus.FieldLogger) *process.Queue {
 
-	manager.DefineStages([]string{"cluster", "check"})
-	manager.AddStep("cluster", update.NewInitialisationStep(db.Instances(), db.Operations(), inputFactory))
-	manager.AddStep("cluster", update.NewUpgradeShootStep(db.Operations(), db.RuntimeStates(), provisionerClient))
-	manager.AddStep("check", update.NewCheckStep(db.Operations(), provisionerClient, 40*time.Minute))
+	updateSteps := []struct {
+		stage     string
+		step      update.Step
+		condition update.StepCondition
```

Comment:
@piotrmiskiewicz there were few things missing in update queue so I modeled it after the provisioning queue